demos/keyboard_nav/index.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demo: Keyboard Navigation</title>
+  <script src="../../blockly_compressed.js"></script>
+  <script src="../../blocks_compressed.js"></script>
+  <script src="../../javascript_compressed.js"></script>
+  <script src="../../msg/js/en.js"></script>
+  <style>
+    body {
+      background-color: #fff;
+      font-family: sans-serif;
+    }
+    h1 {
+      font-weight: normal;
+      font-size: 140%;
+    }
+  </style>
+</head>
+<body>
+  <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
+    <a href="../index.html">Demos</a> &gt; Keyboard Navigation</h1>
+
+  <p>You can enter keyboard navigation mode by <b>shift clicking anywhere on the
+  workspace or on a block.</b> <br>
+
+  You can then use the WASD keys to move around the workspace or click the show
+  traversal button to display an alternative way to navigate the workspace.<br>
+
+  The demo uses <a href="https://en.wikipedia.org/wiki/Tree_traversal#Pre-order_(NLR)">pre
+  order tree traversal</a> as an alternative way to navigate the blocks, connections,
+  and fields on the workspace.<br><br>
+    W: Previous block/field/input at the same level<br>
```

Comment:
The difference between the controls and the traversal demo is confusing. I'd suggest splitting these up into a description of the keyboard controls (including toolbox, insert, and disconnect keys) and then a separate description of the traversal demo that is very clear the button runs that demo.

Reply:
I like the idea of splitting them up but I started writing out the description for all the other keyboard commands and I think it is too much for the page. I was thinking of keeping the basics, but linking to the keyboard navigation documentation in the future. Thoughts?